src/preset_cli/cli/superset/sync/dbt/command.py

```diff
@@ -150,7 +150,7 @@ def dbt_core(  # pylint: disable=too-many-arguments, too-many-locals
         for config in configs["metrics"].values():
             # conform to the same schema that dbt Cloud uses for metrics
             config["dependsOn"] = config["depends_on"]["nodes"]
-            config["uniqueID"] = config["unique_id"]
+            config["uniqueId"] = config["unique_id"]
```

Comment:
This was a bug, but since the metric unique ID was not needed anywhere it was not caught. With the changes in this PR we use the metric ID to traverse the DAG upstream to find parent models.